### PR TITLE
zh-translation: istio.io/content/zh/docs/tasks/observability/distributed-tracing/overview

### PR DESCRIPTION
**Pull request by charili**
ref: https://github.com/servicemesher/istio-official-translation/issues/1305

### CODE REVIEW

File: content/zh/docs/tasks/observability/distributed-tracing/overview/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Istio 利用 [Envoy 的分布式追踪特性](https://www.envoyproxy.io/docs/env
 
 详情请参考 [Zipkin](../zipkin/)，[Jaeger](../jaeger/) 和 [LightStep](/docs/tasks/observability/distributed-tracing/lightstep/) 任务文档来了解 Istio 如何与这些分布式追踪系统协作的。
 
-## 追踪上下文传递
+## 追踪上下文传递{#Trace-context-propagation}
```
Comment (rootsongjc) [inline content/zh/docs/tasks/observability/distributed-tracing/overview/index.md]: 首字母不用大写

File: content/zh/docs/tasks/observability/distributed-tracing/overview/index.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Istio 利用 [Envoy 的分布式追踪特性](https://www.envoyproxy.io/docs/env
 
 详情请参考 [Zipkin](../zipkin/)，[Jaeger](../jaeger/) 和 [LightStep](/docs/tasks/observability/distributed-tracing/lightstep/) 任务文档来了解 Istio 如何与这些分布式追踪系统协作的。
```
Comment (rootsongjc) [inline content/zh/docs/tasks/observability/distributed-tracing/overview/index.md]: 链接前请加 `/zh` 前缀。

**Comment by tanjunchen** [comment]
@charili  @rootsongjc    冲突太多   建议关闭

**Pull request closed without merging**